### DISCUSSION TITLE
Should I use Portal from Ark or Zag?

### DISCUSSION THREAD

**Discussion by gtonussi** [chakra-ui/ark#1437] [General]
Hi, I have just updated Ark from 0.13.1 to 0.15.0 as I was eagerly waiting for a fix on DatePicker's onChange functionality. 

It seems that it has no backward compatibility for Select component, and my current implementation broke.

Checking the new documentation for it, I see that the Ark's Portal component was replaced for Zag's, but there is no reference of installing Zag on "Getting Started".

Is it a recommendation now? Should I install Zag and use Portal from it instead? Are you planning to sunset Ark's Portal? Are they essentially the same?

**Comment by cschroeter** [comment]
Thanks for taking your time to bring this up.
Happy to say this was just an documentation oversight. The docs have been updated accordingly.

Please import Portal from `@ark-ui/react`
